.github/workflows/python.yml

```diff
@@ -141,57 +146,41 @@ jobs:
                       utils/clusters/**
                       benchmarks/results/**
 
-    test-pubsub:
+    test-pubsub-python:
```

Comment:
Not part of the scope of the PR, but can you add a comment that we run it separately because python testing cant handle the load?